tests/functional/test_apple_events.py

```diff
@@ -52,3 +52,79 @@ def test_osx_custom_protocol_handler(tmpdir, pyi_builder_spec):
     with open(logfile_path, 'r') as fh:
         log_lines = fh.readlines()
     assert log_lines and log_lines[-1] == url, 'Invalid arg appended'
+
+
+@skipif_notosx
+@importorskip('PyQt5')
+def test_osx_event_forwarding(tmpdir, pyi_builder_spec):
+    tmpdir = str(tmpdir)  # Fix for Python 3.5
+    app_path = os.path.join(tmpdir, 'dist',
+                            'pyi_osx_event_forwarding.app')
+
+    logfile_path = os.path.join(tmpdir, 'dist', 'events.log')
+
+    if os.path.exists(logfile_path):
+        os.remove(logfile_path)
+    assert not os.path.exists(logfile_path), 'Events logfile still exists!'
```

Comment:
I guess this cannnot be possible. If it exists, the previous line should have deleted it or an exception would be raised and the test would fail, right?

Reply:
That's true.  Was being paranoid. Will delete line.

Reply:
Line deleted in latest commit.